app/create/controller.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+module.exports = function ($scope, Postcard) {
+
+  $scope.loading = false;
+  $scope.error = null;
+  $scope.postcard = null;
+
+  $scope.create = function (payload) {
+
+    $scope.payload = { mail_type: 'usps_first_class' };
```

Comment:
But shouldn't the feature flag for payload only be set when the postcard is being created?